db/migrate/20250224201930_convert_urgency_impact_to_integer.rb

```
@@ -0,0 +1,4 @@
+class ConvertUrgencyImpactToInteger < ActiveRecord::Migration[7.1]
+  def change
+  end
```

RuboCop (1.69.1)

[convention] Put empty method definitions on a single line.

(Style/EmptyMethod)